openslides/__main__.py

```diff
@@ -191,7 +191,8 @@ def start(args):
         'runserver',
         '{}:{}'.format(args.host, args.port),
         '--noreload',
-        '--insecure'])
+        '--insecure',
+        '--nothreading'])
```

Comment:
Can you add a short comment below line 188, please? By the way, the comment is out of date.

Suggestion for the whole comment:
 ```python
# Start the webserver
# Use flag --noreload to tell Django not to reload the server.
# Use flag --insecure to serve static files even if DEBUG is False.
# Use flag --nothreading to tell Django Channels to run in single thread mode.
```